modules/apps/object/object-rest-test/src/testFunctional/macros/CustomObjectAPI.macro

```diff
@@ -203,6 +169,65 @@ definition {
 		return ${response};
 	}
 
```

Comment:
Can you take a look at it? I'm not sure, Is it the correct macro name? 'nObjectEntryBodyAsNestedField'

Reply:
yes, it is correct, the 'n' stands for an undefined number here.